compiler/rustc_type_ir/src/sty.rs

```diff
@@ -18,6 +18,34 @@ use self::TyKind::*;
 use rustc_data_structures::stable_hasher::HashStable;
 use rustc_serialize::{Decodable, Decoder, Encodable};
 
+/// Specifies how a trait object is represented.
+#[derive(Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash, Debug, Encodable, Decodable)]
+pub enum DynKind {
+    /// An unsized `dyn Trait` object
+    Dyn,
+    /// A sized `dyn* Trait` object
+    ///
+    /// These objects are represented as a `(data, vtable)` pair where `data` is a usized value
```

Comment:
```suggestion
    /// These objects are represented as a `(data, vtable)` pair where `data` is a ptr-sized value
```
'usized' looks almost like 'unsized' so I was very confused for a sec ;)

Reply:
Also more accurate, since pointers might not always be the same size as usize :)